packages/sdk/src/assets/nft.validation.ts

```diff
@@ -63,20 +63,13 @@ export const validate = {
 }
 
 export type CreateNFTSchema = z.infer<typeof createNFTSchema>
+export type NFTMetadata = z.infer<typeof metadataSchema>
+export type IpfsInfo = z.infer<typeof ipfsInfoSchema>
 
 export type CreateNFT = {
   contract: HolographERC721Contract | HolographOpenEditionERC721ContractV1 | HolographOpenEditionERC721ContractV2
-  ipfsInfo?: {
```

Comment:
You'll need to revert these changes, this will mess up with the TypeScript intellisense and the user will not be able to get TS autocomplete from this properties unless the user also installs Zod